src/Home.css

```diff
@@ -0,0 +1,243 @@
+  }
+  
+  .weekLabel {
+    background-color: var(--white);
+  
+    display: flex;
+    flex-direction: column;
+    flex-grow: 70;
+    flex-grow: 70;
+  
+    justify-content: space-evenly;
+  
+    height: 72px;
+  }
+  
+  .daysOfWeek {
+    background-color: var(--white);
+    display: flex;
+    flex-direction: row;
+  
+    justify-content: space-around;
+    gap: 5px;
+  }
+  
+  .dateStyle {
+    font-size: 14pt;
+    font-weight: 400;
+    color: var(--darkPurple1);
+  }
+  
+  .timeStyle {
+    font-size: 12pt;
+    font-weight: 400;
+    word-spacing: -5px;
+    color: var(--darkPurple1);
+  }
+  
+  .calendarBodyContainer {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+  
+    align-items: flex-start;
+  }
+  
+  .timeMarkings {
+    background-color: var(--white);
+  
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-right: 10px;
+  
+    width: 54px;
+    /* minus padding */
+    height: 95.5%;
+  
+    text-align: end;
+  }
+  
+  .eventContainer {
+    background-color: var(--purpleEventBg);
+  
+    z-index: 2;
+    border-radius: 4px;
+  
+    grid-column: monStart;
+    grid-row: AM10start / PM1start;
+  
+    display: flex;
+    flex-direction: row;
+  
+    align-items: flex-start;
+    overflow: hidden;
+    overflow: hidden;
+  
+    gap: 6px;
+  }
+  
+  .rect {
+    background-color: var(--purpleEvent);
+    border-radius: 4px 0 0 4px;
+    width: 5px;
+    height: 100%;
+  }
+  
+  .eventInfo {
+    width: 100%;
+    width: 100%;
+    height: auto;
+  
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+  
+    padding-right: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+  
+    font-family: var(--regularFont);
+    font-weight: 300;
+    font-size: 12pt;
+    color: var(--white);
+    transform: scale(1, 0.95);
+  }
+  
+  .eventTopBar {
+    display: flex;
+    flex-direction: row;
+  
+    justify-content: space-between;
+  }
+  
+  .eventTime {
+    display: flex;
+    flex-direction: row;
+  
+    font-size: 11pt;
+  
+    gap: 3px;
+  }
+  
+  .eventName {
+    font-size: 14pt;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+  
+  .calendarGrid {
+    background-color: var(--lightGray);
+  
+    display: grid;
+    width: 100%;
+    height: 100%;
+  
+    grid-template-rows: [AM7start] 1fr [AM730start] 1fr [AM8start]1fr [AM830start] 1fr [AM9start]1fr [AM930start]1fr [AM10start]1fr [AM1030start]1fr [AM11start]1fr [AM1130start]1fr [PM12start]1fr[PM1230start]1fr [PM1start]1fr [PM130start]1fr [PM2start]1fr [PM230start]1fr [PM3start]1fr [PM330start]1fr [PM4start]1fr [PM430start]1fr [PM5start]1fr [PM530start]1fr [PM6start]1fr [PM630start]1fr [PM7start]1fr [PM730start]1fr [PM8start]1fr [PM830start]1fr [PM9start]1fr [PM930start] 1fr [PM10start];
+    grid-template-columns: [sunStart]3fr [monStart]3fr [tueStart]3fr [wedStart]3fr [thuStart]3fr [friStart]3fr [satStart]3fr [weekEnd];
```

Comment:
Instead of defining the rows and columns this way, i recommend using [repeat](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat).